float32 x
float32 y
string algorithm
int32 obstacle_list
---
float64[] pathx
float64[] pathy
